### PR TITLE
#16/team member component

### PR DESCRIPTION
**Pull request by George-Onyango**
# Description

This is pull request is for  a reusable component that allows for the repeated use when rendering the different people in the organization and what their position and description of their journey so far in the organization.

Dependency
Font Awesome

Fixes # (issue) - #16 (https://github.com/italanta/elewa-group/issues/16)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Screenshot
Web View
Screenshot from 2023-02-13 14-56-41

Mobile view

Screenshot from 2023-02-13 14-58-26

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 
Please also list any relevant details for your test configuration if necessary.
- [x] Change media for web to mobile renders as design

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by George-Onyango** [comment]
@IanOdhiambo9 waiting review

File: libs/features/components/banners/src/lib/components/team-member/team-member.component.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+      <div class="desc-wrapper">
+        <p class="p3">{{ descrition }}</p>
+      </div>
+      <div class="desc-wrapper">
+        <p class="p3">{{ descrition }}</p>
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (IanOdhiambo9) [inline libs/features/components/banners/src/lib/components/team-member/team-member.component.html:25]: I don't see any buttons that move to previous and next team member?

File: libs/features/components/banners/src/lib/components/team-member/team-member.component.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  selector: 'elewa-group-team-member',
+  templateUrl: './team-member.component.html',
+  styleUrls: ['./team-member.component.scss'],
+})
+export class TeamMemberComponent {
+  @Input () name = 'Jente Rosseel'
+  @Input () title = 'Founder,Managing Partner'
+  @Input () imageSrc = 'https://i.postimg.cc/6pWDZNNp/jente.jpg'
+  @Input () descrition = 'Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vero quia quae aspernatur similique, adipisci, debitis hic commodi impedit natus exercitationem tempore error, ratione dolore aliquam quis. Officia officiis veniam commodi.'
+  @Input () linkedinUrl = ''
```
Comment (IanOdhiambo9) [inline libs/features/components/banners/src/lib/components/team-member/team-member.component.ts:13]: Please take note that this component is a carousel too as depicted on the design hence takes an array of the data above

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

File: libs/features/components/banners/src/lib/components/team-member/team-member.component.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
...351FIqscA1G0TKBg0aQLYS4DOM0kqqjAAihiJCeeffTJFbIJNGgbErkcXOuoctk0tNQllPO/eOWO5PWpoW4LQY24nJb1AxgU+KECETTYVcZC+dDTTcWRGPDT3fCNWvLI5PtnfrtSE4O1KctjPPpSVQmJnBrh511d0rAONXGjRgsAUyD07tWz8RVOAWOBzNaLSFltySid4SM8PIJ60RZMtGuLfT3QPD4pHwqg9PPG/5Vci9tks2lgyzhSwi5MxHQDrWNZJINRlNzOc4JSQjwnr64qW24pZ4z37oUIZWnIZDvtkYGK554McnZ1Y/KzQjXwBSWWo3csksttIS8hYj2nP50DPBLZ3DRToUbHI9RWz026lubif6XAEKAAcJ6+h+8NjWW1tklv5mDDiVuHBO+OlVjfRGXPJFbz8GcKqnrgc6ZeMkioUztzzvUK4OzE7+lPeNe78PGx55yMAUaBbZNZDDKxAxmrednv7MQQwO8kfjDxnZfUn41W6cqPEQ0KMxXwMy8W/ngmri1vmgEReTOIuF4FACn2Dz91ZmRLbuL+yLOOG8g8MgH3sDn7McqGxknFOuphayDVLUExkIkiHb+HmOXlSlQLghGyh3U/tKdwfhipyVclYyshVTxbmpUy7cNLNbsPEu9Di5ERwRvSBvkLEPC2SaZIFOcmmd/wAYyKEuZiAQMiskwNoVmUEiuoYS7bjeupxSLUrgSNGsZ8AHIUEOdK/MeykFUgqQs5bOxx+7joKb1pxpOtEA2lrjSUQEkOe8yqliPStKLw6c0YmjIMgBV+a5wDuDvyNVGkcmqbW/7tbfP/bSj8CPlhUGY9R72RQVkZjJjcBTyBHPHtFXh0WMwk2ihzKAGjVwRw55jPs86o4/8Ms/9RrT2H6s/wCn/wCGpMg+PsCsYreCykSOR2SN+Euq5zITvj2bD41jdVXh1G4HEW8Z3xitVpH+HJ+9/I1ldR/vk/7xqy7MDqedSidTGEkXHDsCMjIqEda7pTARZ6cInh+t74cJweA7e+riO3tkVDEAN9idziqTTf1U3sq7T+7wf+v4igZk9/AJ9Pu4RvmMuMftLvt7aqY3dbe1YAbwKDj3/lirqP8AUv8AuW/Cqn/wrX9yv4Uk+hoBUV2vBhjvigpu6kk4mNRLzaoH50iQ7Yazqq+EihpZA+xxmmryqH79FGkx7KM7UlOrqYU//9k=',
```
Comment (IanOdhiambo9) [inline libs/features/components/banners/src/lib/components/team-member/team-member.component.ts:19]: This image address is too long, I'm sure you can find a shorter one on unsplash or pexels etc

File: libs/features/components/banners/src/lib/components/team-member/team-member.component.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+previousMember() {
+  this.currentMemberIndex = (this.currentMemberIndex - 1 + this.members.length) % this.members.length;
+}
+
+get whichMember () {
+  return (this.currentMemberIndex + 1) / this.members.length*100;
+}
+
+closeMemberView() {
+  console.log('return to about page')
```
Comment (IanOdhiambo9) [inline libs/features/components/banners/src/lib/components/team-member/team-member.component.ts:40]: remove code that is not been used

File: libs/features/components/banners/src/lib/features-components-banners.module.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { TeamMemberComponent } from './components/team-member/team-member.component';
+
+@NgModule({
+  imports: [CommonModule],
+  declarations: [TeamMemberComponent],
+})
+export class FeaturesComponentsBannersModule {}
```
Comment (IanOdhiambo9) [inline libs/features/components/banners/src/lib/features-components-banners.module.ts:9]: Rename to just banners module

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Add screenshots of your work on the PR template and a short video showing the carousel working on mobile and web

File: libs/features/components/banners/src/lib/components/team-member/team-member.component.ts (hunk 4 above)
Comment (George-Onyango) [reply libs/features/components/banners/src/lib/components/team-member/team-member.component.ts:40]: It is used when you want to close the view to go back to the team member carousel

**Comment by IanOdhiambo9** [comment]
I'm closing this PR because you have a package.json file. Do a new PR without this file.

**Pull request closed without merging**